packages/electron/scripts/docker-make.sh

```
@@ -155,7 +155,20 @@ if [ "$PLATFORM" = "linux" ]; then
+  # app dir. The AppImage + latest-linux.yml + app-update.yml come from
+  # electron-builder --prepackaged (Forge AppImage maker removed).
+  # See change: fix-electron-auto-update-pipeline (D1).
   ../../node_modules/.bin/electron-forge make --platform linux --arch "$ARCH"
+  PKG_DIR=$(find out -maxdepth 1 -type d -name 'PI-Dashboard-linux-*' | head -1)
+  if [ -z "$PKG_DIR" ]; then
+    echo "::error:: Forge packaged dir not found under out/"; ls -la out || true; exit 1
+  fi
+  ../../node_modules/.bin/electron-builder --linux AppImage \
+    --prepackaged "$PKG_DIR" \
+    --config electron-builder.yml \
+    -c.linux.artifactName='PI-Dashboard-${version}-'"$ARCH"'.${ext}' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)